web-www/components/templates/shared/EmailSubmitSection/EmailSubmitSection.tsx

```diff
@@ -0,0 +1,67 @@
+import clsx from 'clsx';
+
+import { BlockContent } from 'web-components/lib/components/block-content';
+import NewsletterForm from 'web-components/lib/components/form/NewsletterForm';
+import { Label, Title } from 'web-components/lib/components/typography';
+
+import { useEmailSubmitSectionStyles } from './EmailSubmitSection.styles';
+
+import { BackgroundImage } from '@/components/organisms/BackgroundImage/BackgroundImage';
+import { SharedNewsletterSectionFieldsFragment } from '@/generated/sanity-graphql';
+
+interface Content {
+  header?: string;
+  description?: string;
+  buttonText?: string;
+  inputText?: string;
+}
+
+type Props = {
+  altContent?: Content;
+  sharedNewsletterData?: SharedNewsletterSectionFieldsFragment['newsletter'];
+};
+
+const EmailSubmitSection = ({ altContent, sharedNewsletterData }: Props) => {
+  const { classes: styles } = useEmailSubmitSectionStyles();
+  const content = sharedNewsletterData;
+  return (
+    <BackgroundImage
+      src="/images/token/deer-newsletter-bg.png"
+      component="section"
+      sx={{ background: '#040e18' }}
+    >
+      <div className={clsx(styles.root)} id="newsletter-signup">
+        <Title
+          variant="h2"
+          mobileVariant="h3"
+          sx={{ color: 'primary.main', textAlign: 'center' }}
+        >
+          {altContent?.header || content?.title}
+        </Title>
+        <Label
+          as="div"
+          sx={{
+            color: 'primary.main',
+            textAlign: 'center',
+            pt: [5, 7.5],
+            pb: [4, 6.25],
+          }}
+        >
+          {altContent?.description ? (
+            altContent.description
+          ) : (
+            <BlockContent content={content?.bodyRaw} />
+          )}
+        </Label>
+        <NewsletterForm
+          apiUri={process.env.GATSBY_API_URI}
```

Comment:
should we add .env and netlify.toml config for that?
this form is not working as is

Reply:
I have to migrate these env variables yes, will do that.

Reply:
The env file is already set up and the `netlify.toml` defined in `regen-web` already has everything we need, it's just about renaming these env variables.

Reply:
right, we should get rid of `GATSBY_` prefix which doesn't make sense anymore 😅